test.js

```diff
@@ -70,3 +70,11 @@ it('should not mutate the options object - async', function () {
 it('should not mutate the options object - sync', function () {
 	globby.sync(['*.tmp', '!b.tmp'], Object.freeze({ignore: Object.freeze([])}));
 });
+
+it('should expose generateGlobTasks', function () {
+	var tasks = globby.generateGlobTasks(['*.tmp', '!b.tmp'], {ignore: ['c.tmp']});
+
+	assert.strictEqual(tasks.length, 1);
+	assert.strictEqual(tasks[0].pattern, '*.tmp');
+	assert.deepEqual(tasks[0].opts.ignore.sort(), ['b.tmp', 'c.tmp']);
```

Comment:
The `.sort()` implies that the order is non-important and/or non-deterministic. I guess that may be so, but I'd normally expect the passed in `ignore` option patterns to precede the ones added by globby's implementation (generateGlobTasks).